README.md

```diff
@@ -119,31 +118,40 @@ docker-compose exec rails bash
 
 ### Running tests
 
-To run the core tests:
+#### Hitobito core
+
+To run all tests:
+
+```bash
+bin/test_env_core bundle exec rspec
+docker-compose run rails_test_core bundle exec rspec
+
+```
+
+To run specific tests:
+
 ```bash
-# Open a testing console in the container
-docker-compose exec -e RAILS_ENV=test rails bash
-# Run all the specs
-rspec
-# Run specs from a single file
-rspec spec/models/person_spec.rb
+bin/test_env_core bundle exec rspec spec/models/person_spec.rb
```

Comment:
maybe always use `bin/test_env_core bash` for running tests since like this auto completion is not working.

we could also consider adding bundle exec bash directly to bin/test_env_core
so just call `bin/test_env_core` without any arguments to get a test console

Reply:
also check comment for `bin/test_env_core` further down

Reply:
Makes sense, will do